server/src/routes/api/v1/communities.js

```diff
@@ -263,10 +263,10 @@ router.get('/:communityAddress/:accountAddress', async (req, res) => {
   if (lodash.has(community, 'plugins.onramp.services.transak.widgetUrl') && config.has('plugins.transak.api.secret')) {
     community.plugins.onramp.services.transak.widgetUrl = `${community.plugins.onramp.services.transak.widgetUrl}&partnerCustomerId=${accountAddress}_${toChecksumAddress(communityAddress)}`
   }
-  // Todo - handle externalCustomerId
-  // if (lodash.has(community, 'plugins.onramp.services.rampInstant.widgetUrl')) {
-  // community.plugins.onramp.services.rampInstant.widgetUrl = `${community.plugins.onramp.services.rampInstant.widgetUrl}&userAddress=${accountAddress}_${toChecksumAddress(communityAddress)}`
-  // }
+  if (lodash.has(community, 'plugins.onramp.services.rampInstant.widgetUrl') && config.has('plugins.rampInstant.api.secret')) {
+    const webhookStatusUrl = `${config.get('api.protocol')}://${req.headers.host}/api/v1/deposits/ramp/${accountAddress}_${toChecksumAddress(communityAddress)}`
```

Comment:
I think you can do just /accountAddress/communityAddress. so no need to split it/combine